### DISCUSSION TITLE
separate channels?

### DISCUSSION THREAD

**Discussion by ghost** [igece/SoxSharp#27] [Q&A] [answered]
It looks like I'm able to customise some arguments to the sox command by passing fake input filenames, but how can I pass arguments, e.g. `remix 1` _after_  the output file?

E.g. replicate this command

    sox file.wav -b 16 -r 16k output.wav remix 1

**Comment by igece** [comment, marked as answer]
To pass customized arguments you should use the `CustomArgs` property of a `Sox` instance. However, what you want is to use a still unimplemented effect (`remix`), so in this case the property you should use is `CustomEffects`, setting it to "remix 1".
The difference between `CustomArgs` and `CustomEffects` properties is that the content of the first one is placed before any input argument, and the content of the second one is placed after all the output  arguments (where effects are passed).
